discoverable-client/src/main/resources/application.yml

```diff
@@ -138,6 +138,9 @@ apiml:
                 lb:
                     type: headerRequest
 
+        connectTimeout: 10
```

Comment:
Why these values are different than those in ApiMediationServiceConfig class? Do we provide so small timeouts (probably the original one)